components/script/dom/globalscope.rs

```diff
@@ -231,6 +235,20 @@ struct TimerListener {
     context: Trusted<GlobalScope>,
 }
 
+///  A wrapper for blob data sent over the ipc router.
```

Comment:
I think the below could be a more correct description:
`A wrapper for the handling of file data received by the ipc router`